MVVM Module/Source/Feed UI/Controllers/FeedViewController.swift

```diff
@@ -23,6 +23,10 @@ public final class FeedViewController: UITableViewController, UITableViewDataSou
 		viewModel?.loadFeed()
 	}
 
+	private var errorView: ErrorView? {
+		tableView.tableHeaderView as? ErrorView
+	}
```

Comment:
I recommend creating an IBOutlet instead of extracting the error view from the `tableHeaderView`.

Ideally, the view controller should be decoupled from the view hierarchy in the storyboard.

This way, you can easily change the view hierarchy in the storyboard without breaking the code here.
```suggestion
	@IBOutlet private(set) var errorView: ErrorView?
```